app/src/main/kotlin/com/intive/picover/auth/repository/AuthRepository.kt

```diff
@@ -30,6 +30,15 @@ class AuthRepository @Inject constructor(
 		firebaseAuth.signOut()
 	}
 
+	fun photoUrl() =
+		firebaseAuth.currentUser!!.photoUrl
+
+	fun displayName() =
+		firebaseAuth.currentUser!!.displayName
+
+	fun email() =
+		firebaseAuth.currentUser!!.email
```

Comment:
Shouldn't we use coroutines here ?

Reply:
Done 😄

Reply:
it was just a question that did not suggest anything .... :D

Reply:
Ok, I know it but when I considered this option I would like to try to use coroutines here :D